tools/packaging/common/envoy_bootstrap_v2.json

```diff
@@ -333,7 +333,9 @@
               ],
               "validation_context": {
                 "trusted_ca": {
-                  {{- if eq .pilot_cert_provider "kubernetes" }}
+                  {{- if .provisioned_cert }}
+                  "filename": "{{(printf "%s%s" .provisioned_cert "/root-cert.pem") }}"
```

Comment:
nit: can you change 
```
pkg/test/framework/components/echo/docker/workload.go
154:                    "PROV_CERT="+istio_agent.CitadelCACertPath,
```

To use a different path? Then we can appropriately test that you can use something other than `./var/run/secrets/istio`

Alternatively, do we even **want** to support other locations?

Reply:
with this change, we can let user define any location they like.